services/brig/src/Brig/App.hs

```diff
@@ -441,6 +442,10 @@ providerTemplates l = forLocale l <$> view provTemplates
 teamTemplates :: (MonadReader Env m) => Maybe Locale -> m (Locale, TeamTemplates)
 teamTemplates l = forLocale l <$> view tmTemplates
 
+-- this works because team templates is not affected by `forLocale`
```

Comment:
```suggestion
-- this works because team templates is not affected by `forLocale`; it is useful where we
-- use the `TeamTemplates` only for finding invitation url templates (those are not localized).
```